src/bqlfn.py

```diff
@@ -280,6 +281,35 @@ def bql_column_value_probability(bdb, generator_id, modelno, colno, value,
     r = metamodel.logpdf_joint(
         bdb, generator_id, targets, constraints, modelno)
     return ieee_exp(r)
+
+# XXX This is silly.  We should return log densities, not densities.
```

Comment:
When I ask why, I'm in large part predicting what other people will ask. Rather than answering me in line notes here, may I ask you to add that pointer, and a bit more context, to the XXX note?